openfecwebapp/templates/legal-search-results.html

```diff
@@ -33,17 +40,18 @@
           </ul>
         </nav>
       </div>
-      <section class="main__content">
+      <section class="main__content--right">
+        {% if query %}
+        <h1 class="main__title">Results for “{{ query }}”</h1>
```

Comment:
These quotes are smartquotes (which they should be) but should be replaced by the HTML entities.